content/docs/reference/design-docs/train-sim-v3/driver-behavior-modules.en.md

```diff
@@ -3,36 +3,80 @@ title: "Driver behavior modules"
 weight: 30
 ---
 
-### Driver behavior modules
 
-```mermaid
-flowchart TD
+## Design specs
 
-subgraph BuilderInputs["builder inputs"]
-    SlowdownCoefficient[slowdown coefficient]
-    PathProps[path properties]
-end
+### General pitch
 
-subgraph CallInputs["call inputs"]
-    ConcreteDrivingInstructions[concrete driving instruction]
-    CurrentTrainState[current train state]
-end
+Driver behavior modules are responsible for making driving decisions. 
+It's main responsibility, given the state of the train and an instruction, is to react to the instruction.
+This reaction is expressed as a new train state.
 
-DriverBehaviorModule[driver behavior module]
-Decision[decision]
+To perform this critical task, it needs access to additional context:
 
-BuilderInputs --> DriverBehaviorModule
-CallInputs --> DriverBehaviorModule --> Decision
+- the physical properties of the path, which are used to make coasting decisions, and to model natural forces.
+- a slowdown kind, dictating how time loss is distributed along the path
```

Comment:
the slowdown kind _is_ the driver behavior module implementation

Reply:
I know, I just didn't know how to express that
I'll try something a bit better